backend/grant/user/views.py

```diff
@@ -269,12 +269,12 @@ def delete_avatar(url):
 @blueprint.route("/<user_id>", methods=["PUT"])
 @auth.requires_auth
 @auth.requires_same_user_auth
-# TODO gaurd all (minimum, minimum, shape, uri)
 @body({
-    "displayName": fields.Str(required=True),
-    "title": fields.Str(required=True),
+    "displayName": fields.Str(required=True, validate=lambda d: 2 <= len(d) <= 200),
+    "title": fields.Str(required=True, validate=lambda t: 2 <= len(t) <= 200),
```

Comment:
I think we limit these on the frontend to something much smaller, 60 chars maybe?